content/docs/guides/contribute/review-process.en.md

```diff
@@ -0,0 +1,20 @@
+Here's a suggested workflow. The quotation marks indicate the **necessity** of communication to ensure the smooth flow of PR validation:
+
+```mermaid
+sequenceDiagram
+  actor A as PR author
+  actor R as Reviewer/Maintainer
+  
+  A->>R: ASKS "I need a review"
+  R->>A: ANSWERS ✅ "Ok" or ❌ "not ok"
```

Comment:
```suggestion
  R->>A: ANSWERS ✅ Approves or ❌ Requests changes
```

Reply:
I suggest `R->>A: ANSWERS ✅ "Ok, I'll do the review" or ❌ "not ok, I can't do it"`

Reply:
Is it an interaction even worth mentioning? someone will review it at some point

Reply:
Same response as above.